src/components/Stepper/MuiStepper/MuiStepper.tsx

```diff
@@ -11,14 +11,15 @@ import {
   
  
   
-  const OnboardingStepIcon: React.FC<StepIconProps> = ({ active, completed }) => {
+  const OnboardingStepIcon: React.FC<StepIconProps & { paletteColors: any }> = ({ active, completed, paletteColors }) => {
```

Comment:
We've talked about this but it seems like this paletteColors can't use dynamically based on your code here we just added a new type but the usage is not reachable outside of our stepper component. You can remove this paletteColors and just drop the required colors based on wireframe. 

Once changed : this will apply to all our stepper component already implemented.